File: src/main/java/problems/MiniMaxSum.java
package problems;

import java.util.Scanner;
import java.util.stream.IntStream;

public class MiniMaxSum {
    static void miniMaxSum(int[] arr) {
        long sum = IntStream.of(arr).sum();
        long min = sum - IntStream.of(arr).summaryStatistics().getMax();
        long max = sum - IntStream.of(arr).summaryStatistics().getMin();
        System.out.println(min + " " + max);
    }

    private static final Scanner scanner = new Scanner(System.in);

    public static void main(String[] args) {
        int[] arr = new int[5];

        String[] arrItems = scanner.nextLine().split(" ");
        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");

        for (int i = 0; i < 5; i++) {
            int arrItem = Integer.parseInt(arrItems[i]);
            arr[i] = arrItem;
        }

        miniMaxSum(arr);

        scanner.close();
    }
}

File: js/mini-max-sum.js
function miniMaxSum(arr) {

    const sum = arr.reduce((a, b) => a + b);
    const min = sum - Math.max(...arr);
    const max = sum - Math.min(...arr);
    console.log(min, max);

}

myArray = [1, 2, 3, 4, 5];
miniMaxSum(myArray);
